docs/source/sections/webapp/Saved-Cohorts.rst

```diff
@@ -392,33 +402,48 @@ platforms by dragging the platform names up and down.
+   +-----------------------------------------+------------------------------------------+-----------------------------------------+-----------------------------------------+
+   | Sample Barcode                          | Program                                  | Program                                 | Program                                 |
+   +-----------------------------------------+------------------------------------------+-----------------------------------------+-----------------------------------------+
+   | Platform                                | Platform                                 | Platform                                | Platform                                |
+   +-----------------------------------------+------------------------------------------+-----------------------------------------+-----------------------------------------+
+   | Pipeline                                | Experimental Strategy                    | Experimental Strategy                   | Experimental Strategy                   |
+   +-----------------------------------------+------------------------------------------+-----------------------------------------+-----------------------------------------+
+   | Data Level                              | Data Category                            | Data Category                           | Data Category                           |
+   +-----------------------------------------+------------------------------------------+-----------------------------------------+-----------------------------------------+
+   | File Path to the Cloud Storage Location | Data Type                                | Data Type                               | Data Type                               |
+   +-----------------------------------------+------------------------------------------+-----------------------------------------+-----------------------------------------+
+   | Access type (open or controlled access) | Data Format                              | Data Format                             | Data Format                             |
+   +-----------------------------------------+------------------------------------------+-----------------------------------------+-----------------------------------------+
+   |                                         | File Path to the Cloud Storage Location  | File Path to the Cloud Storage Location | File Path to the Cloud Storage Location |
+   +-----------------------------------------+------------------------------------------+-----------------------------------------+-----------------------------------------+
+   |                                         | Access type (open or controlled access)  | Access type (open or controlled access) | Access type (open or controlled access) |
+   +-----------------------------------------+------------------------------------------+-----------------------------------------+-----------------------------------------+
+
 
```

Comment:
@madelyngreyes The File Path and Access type are also in the downloads from the All Files tab. The download is identical for all tabs; the same process runs each one, so there's no difference in the columns.